Lib/test/test_generators.py

```diff
@@ -170,6 +170,26 @@ def f():
             g.send(0)
         self.assertEqual(next(g), 1)
 
+    def test_handle_frame_object_in_creation(self):
```

Comment:
Can you also add the [`__del__` test here](https://github.com/python/cpython/issues/94262#issuecomment-1167651371 )?

Reply:
> Whether the code run is inspect.stack() or a finalizer doesn't really make any difference.


I don't think so.

The code from the issue still segfaults for me on this branch on Linux:

```py
import gc
import inspect

gc.set_threshold(1, 0, 0)
f = []

def cb(*args):
    f.append(inspect.stack())

gc.callbacks.append(cb)

def gen():
    yield 1

g = gen()
g.__next__()
```